tasks/build-binary-new/builder.rb

```diff
@@ -714,8 +714,13 @@ def execute(binary_builder, stack, source_input, build_input, build_output, arti
         raise "Unsupported jruby version line #{source_input.version}"
       end
 
+
       # Create a copy of the source_input to prevent mutating version for later use
       full_version = "#{source_input.version}-ruby-#{ruby_version}"
+
+      filename = "#{binary_builder.base_dir}/#{source_input_name}-#{full_version}-linux-x64.tgz"
+      Archive.strip_incorrect_words_yaml_from_tar(filename)
```

Comment:
The strip must occur after the function `binary_builder.build` is called on line 724, otherwise the .tgz does not exist yet.